### ISSUE TITLE
Decoded images ignore image orientation

### ISSUE THREAD

**Issue by waltflanagan** [pinterest/PINRemoteImage#39]
Using UIImageView `-pin_setImageFromURL:` and related methods lose orientation metadata and display incorrectly.  

This seems to be because that are decoded using `pin_decodedImageWithCGImageRef:` don't take the images orientation into account when drawing the CGImageRef into the context. 

Repro steps:
```swift 
imageView.pin_setImageFromURL(NSURL(string: "https://www.dropbox.com/s/nakmlzyqkqd8ie8/04cd607b-d5ba-4f4a-9344-77c8ef94c291--2015-08-12-08-41-27.jpg?dl=1"))
```

That image was taken with an iPhone 6, portrait orientation.  It appears rotated when set on the imageview via `pin_setImageFromURL`.  Viewing it in a browser, preview, etc show it correctly.

**Comment by garrettmoon** [comment]
Awesome, thank you for the pull request!

**Comment by garrettmoon** [comment]
Fixed by #40
